<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visor DICOM</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="campo_grafico" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>561</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_cargar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cargar Imagen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_atras">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_adelante">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_aumentar">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aumentar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_disminuir">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Disminuir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>430</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Desplazamiento entre cortes</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="convD">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>430</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convertir a DICOM</string>
    </property>
   </widget>
   <widget class="QLabel" name="id_paciente">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>120</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sexo del paciente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>210</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha nacimiento</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>90</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Identificaci√≥n del paciente</string>
    </property>
   </widget>
   <widget class="QLabel" name="medico_tratante">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>360</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NA</string>
    </property>
   </widget>
   <widget class="QLabel" name="fecha_examen">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>180</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre del paciente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>330</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Medico Tratante</string>
    </property>
   </widget>
   <widget class="QLabel" name="fecha_nacimiento">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>240</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NA</string>
    </property>
   </widget>
   <widget class="QLabel" name="nombre_paciente">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>60</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de examen</string>
    </property>
   </widget>
   <widget class="QLabel" name="sexo_paciente">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>300</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NA</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
